wiki/platform_details.md

```diff
@@ -38,19 +38,28 @@ Each Learning Module will be no longer than 10 pages of content ( with pictures/
 ### 2. Home
 - Summary of achievements (score, streaks)
 - Icon for each Learning Module, shaded if completed, colorful if open, transparent if locked
+  with a percentage icon showing the total amount completed for the learning module
 - Link to profile settings
 - Link to Leadership board
 
 ### 3. Profile settings
+- Show or hide email, mobile number for Telegram/Whatsapp , Twitter/Github/Facebook handle
+- Add profile picture
+- Tell us more about yourself
+- Goals and aspirations
+- Tech stack
 
 ### 4. Learning Modules 
 - Go to previous or next slide
+- Add a pomodoro timer (with time selector for focus time and break time e.g. 25 minutes focus countdown, 5 minute break
```

Comment:
Will it be necessary to have a timer if they are going through slides, which is an already paced format?

Reply:
Hi @VirginiaBalseiro ,

The Pomodoro timer/method is a nice focus time - keeps you motivated and focused, I use it daily in my studies. It can be an overall timer above the normal slides, to tell the user he has finished 25 minutes and he can take a 5-minute break. For 3x 25 minute sessions you get 3x 5min breaks in between, and then you get a 40 min break after the 4th break. Makes it easier to stay committed . I use an app called Forest in my studies , and implement the Pomodoro method.

Let me know what you think about it, 
Thanks! 😊 🙏

Reply:
The idea sounds indeed great and helpful, but let's add a "Nice to have: " in front so we can keep it for a later round. Let's focus on building the minimum features on this round.